docs/content/connectors/mysql-cdc.md

```diff
@@ -392,6 +392,25 @@ public class MySqlBinlogSourceExample {
 
 **Note:** Please refer [Deserialization](../about.html#deserialization) for more details about the JSON deserialization.
 
+### Well-Supported to MySQL high available cluster
```

Comment:
we can make this following `#### Lock-free` section as this is one of key feature we supported

Reply:
How about the feature name `MySQL High Availability Support` ?